api/python/provisioner/commands/auto_deploy.py

```diff
@@ -106,8 +106,8 @@ def run(self, nodes, **kwargs):
             setup_ctx.ssh_client.cmd_run(
                 (
                     'salt-call state.apply '
-                    'components.system.config.pillar_encrypt'
-                ), targets=setup_ctx.run_args.primary.minion_id
+                    'components.system.config'
```

Comment:
pillar_encrypt here was called here to enrypt updated pillar once config ini is applied.
The whole system.config appliance would trigger [many other routines](https://github.com/Seagate/cortx-prvsnr/blob/6a8e2ce4a47f40dc55851dba71d75823819106d9/srv/components/system/config/init.sls#L19-L24).

More over later dual deploy will call the same but [after prepare and install](https://github.com/Seagate/cortx-prvsnr/blob/6a8e2ce4a47f40dc55851dba71d75823819106d9/srv/components/system/init.sls#L19-L21).

So, that change doesn't look clean. Could you clarify?
CC @ypise

Reply:
Actually I want to run components.system.config.hosts , @ypise suggested instead of running config.pillar_encrypt and config.hosts in 2 steps, it is safe to run complete system.config at this stage. so added complete system.config here.

Reply:
`So, that change doesn't look clean. Could you clarify?`
Yes it does and needs to be worked on. We need to split system based on the CES flow.

Reply:
comment is addressed